### PR TITLE
Sept tmit

### PR DESCRIPTION
**Pull request by Hoverbear**
### What is changed?

Add the september issue of tmit

### CODE REVIEW

**Comment by Hoverbear** [comment]
@zhangjinpeng1987 @AndreMouche @zhouqiang-cl could you take a look and let me know if there is any other notable PRs/issues which I missed?

File: content/blog/monthly-september-19.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+title: This Month in TiKV - September 2019
+date: 2019-10-18
+author: Ana Hobden
+---
+
+Wonderful day! Welcome to the third edition of 'This Month in TiKV', covering September 2019.
+
+## News
+
+This month our team made one TiKV minor release! This minor releases include bug fixes and minor, backwards compatible features.
```
Comment (dcalvin) [inline content/blog/monthly-september-19.md]:
```suggestion
This month our team made one TiKV minor release (v3.0.4)! This release includes bug fixes and minor, backwards compatible features.
```

Comment (Hoverbear) [reply content/blog/monthly-september-19.md]: Oh, thanks xD My english was bad there.

**Comment by zhouqiang-cl** [comment]
LGTM

File: content/blog/monthly-september-19.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Upgrading? Please take note of these things:
+
+* There is a new batch split region command.
+* There is a new `grpc-memory-pool-quota` configuration option.
+
+## Reading materials
+
+Here are some articles our contributors have published over the last month:
+
+* [@AndreMouche] published an article teaching about [TiKV's MVCC implementation (Chinese)](https://pingcap.com/blog-cn/tikv-source-code-reading-13/).
```
Comment (AndreMouche) [inline content/blog/monthly-september-19.md]: @breeswish is the author.

File: content/blog/monthly-september-19.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* [@nrc] talked on his blog about [migrating projects from Futures 0.1 to 0.3](https://www.ncameron.org/blog/migrating-a-crate-from-futures-0-1-to-0-3/).
+* [@yiwu-arbug] and [@pentium3] write about automatically tuning TiKV with [AutoTiKV (Chinese)](https://pingcap.com/blog-cn/autotikv/).
```
Comment (AndreMouche) [inline content/blog/monthly-september-19.md]: only for September? The date is 2019-10-18, and we also push 2 documents about best practice for optimistic transaction and PD in October:
https://pingcap.com/blog-cn/best-practice-optimistic-transaction/
https://pingcap.com/blog-cn/best-practice-pd/

File: content/blog/monthly-september-19.md (hunk 2 above)
Comment (Hoverbear) [reply content/blog/monthly-september-19.md]: Oh no, sorry xD

File: content/blog/monthly-september-19.md (hunk 3 above)
Comment (Hoverbear) [reply content/blog/monthly-september-19.md]: Yes only September! We must add the october documents to the october post. :)

**Pull request merged**